app/assets/stylesheets/_related-navigation.scss

```diff
@@ -1,23 +1,23 @@
 .app-related-navigation {
-  @include govuk-text-colour;
+  @include all.govuk-text-colour;
```

Comment:
Ooooh, what’s this? Is this syntax specific to the new bundling tool?

Reply:
Module specific syntax. @use vs @import https://sass-lang.com/documentation/at-rules/use